src/NuGet/Microsoft.Net.Compilers.netcore.nuspec

```diff
@@ -60,6 +60,8 @@
         </dependencies>
     </metadata>
     <files>
+        <file src="$emptyDirPath$/_._" target="ref/dotnet" />
```

Comment:
The change looks correct assuming those are the right TFM's you need. What is prompting this change?